packages/boat/src/modules/mod/logger.ts

```diff
@@ -94,12 +94,11 @@ async function messageDeleteBulk (this: CommandClient, msgs: Array<Message<Guild
     return // Let's just ignore
   }
 
-  const list = msgs.map(
-    (msg) =>
-      'author' in msg
-        ? format(LIST_TEMPLATE, msg)
-        : `A message in #${msg.channel.name} that was not cached`
-  ).join('\n\n')
+  const list = (await Promise.all(msgs.map(
+    (msg) => 'author' in msg
+      ? format(LIST_TEMPLATE, msg, true)
+      : `A message in #${msg.channel.name} that was not cached`
+  ))).join('\n\n')
```

Comment:
```suggestion
  const list = []
  for (const msg of msgs) {
    list.push('author' in msg
      ? format(LIST_TEMPLATE, msg, true)
      : `A message in #${msg.channel.name} that was not cached`)
  }
```
+ necessary changes in other places now that list is an array

Reply:
Any reason not to use Array.map like the below snippet and then handle the processes elsewhere?
```js
const list = msgs.map((msg) =>
    'author' in msg
      ? format(LIST_TEMPLATE, msg, true)
      : `A message in #${msg.channel.name} that was not cached`)
// Code that handles promises.
```

Reply:
A bit less clean; I actually forgot the await in front of format in my suggestion, that's updated